src/Resources/skeleton/command/Command.tpl.php

```diff
@@ -13,6 +13,11 @@ class <?= $class_name; ?> extends Command
 {
     protected static $defaultName = '<?= $command_name; ?>';
 
+    /**
+    * @var SymfonyStyle
+    */
+    private $io;
```

Comment:
Should this property be private?  If a user extends from the command, will they still have access to the `$this->io` property?